relay_test.go

```diff
@@ -285,8 +288,10 @@ func TestErrorFrameEndsRelay(t *testing.T) {
 // Trigger a race between receiving a new call and a connection closing
 // by closing the relay while a lot of background calls are being made.
 func TestRaceCloseWithNewCall(t *testing.T) {
-	// TODO: enable framepool checks
-	opts := serviceNameOpts("s1").SetRelayOnly().DisableLogVerification()
+	opts := serviceNameOpts("s1").
+		SetRelayOnly().
+		DisableLogVerification().
+		SetCheckFramePooling()
```

Comment:
optional nit: for consistency, suggest putting `SetRelayOnly()` then `SetCheckFramePooling()` in the options next to each other (since they're mostly related in these tests) and ideally in the same order for all tests that use both.